en/qgc/video_streaming_wifi_broadcast.md

```diff
@@ -1,23 +1,26 @@
-# Long-distance Video Streaming in QGroundControl
+# Long-distance Video Streaming and telemetry via raw WiFi radio
 
-This page shows how to set up a companion computer with a camera (Logitech C920 or RaspberryPi camera) such that the video stream is transferred from the UAV to a ground computer and displayed in *QGroundControl*. This setup uses WiFi in unconnected (broadcast) mode and software from the [Wifibroadcast project](https://github.com/svpcom/wifibroadcast/wiki).
+This page shows how to set up a companion computer with a camera (Logitech C920 or RaspberryPi camera) such that the video stream is transferred from the UAV to a ground computer and displayed in *QGroundControl*. Also it provides support for bidirectional telemetry like SiK radio does. This setup uses WiFi in unconnected (broadcast) mode and software from the [Wifibroadcast project](https://github.com/svpcom/wifibroadcast/wiki).
 
 
 ## Wifibroadcast Overview
 
-The *Wifibroadcast project* aims to mimic the advantageous properties of using an analog link to transmit HD video (and other) data when using WiFi radios. For example, it attempts to provide a video feed that degrades gracefully with signal degradation/distance.
+The *Wifibroadcast project* provides video and telemetry transport which use low-level WiFi packets to avoid distance and latency limitations of ordinary ieee80211 stack.
```

Comment:
ieee80211 should be: IEEE 802.11
If you can provide a link to the standard that would be extra cool.